src/browser/api/external_window.ts

```diff
@@ -244,11 +245,16 @@ export function getRegisteredExternalWindow(identity: Identity): Shapes.External
 }
 
 /*
-  Checks whether an hwnd is a valid target for wrapping
+  Checks whether a uuid is a valid target for wrapping
 */
-function doesExternalWindowExist(uuid: string): boolean {
+function findExistingNativeWindow(uuid: string): Shapes.NativeWindowInfoLite|undefined {
   const allNativeWindows = electronApp.getAllNativeWindowInfo(true);
-  return !!allNativeWindows.find(win => getNativeWindowInfoLite(win).uuid === uuid);
+  // Check whether we are using a uuid generated by `launchExternalProcess`
+  const prelaunchedProcess = ProcessTracker.getProcessByUuid(uuid);
+  const win = prelaunchedProcess
+    ? allNativeWindows.find(win => win.process.pid === prelaunchedProcess.process.id)
```

Comment:
Small naming thing, but you're using just `win` here a lot. Maybe specify what each `win` is supposed to be?